deepreg/model/layer.py

```diff
@@ -201,7 +214,17 @@ def __init__(
         self._norm = Norm()
         self._act = Activation()
 
+    def build(self, input_shape):

```

Comment:
Why we add this function? I think we won't need it? The build will be automatically called during the first time of `call`

Reply:
That's right sorry!